sys/linux/socket_netlink_route_386.const

```diff
@@ -432,6 +433,7 @@ NUD_PERMANENT = 128
 NUD_PROBE = 16
 NUD_REACHABLE = 2
 NUD_STALE = 4
+# RPL_IPTUNNEL_SRH is not set
```

Comment:
@dvyukov @xairy How can we get these?

How can vendor/golang.org/x/sys/unix/ztypes_linux.go be updated?

Reply:
We need to add `include "uapi/linux/rpl_iptunnel.h"` and re-run `make extract`.
This is not related to the Go pacakges.